### DISCUSSION TITLE
CreateWebHostBuilder > Exception Unhandled

### DISCUSSION THREAD

**Discussion by alieryaman** [mixcore/mix.core#383] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/26028678/116005992-09dc8080-a612-11eb-85d8-f6fa0734a8f9.png)

**Comment by Smilefounder** [comment, marked as answer]
Hi @alieryaman could you check again if the https://github.com/mixcore/mix.core/blob/master/src/Mix.Cms.Web/default.appsettings.json file is existing?

Remember to remove any appsettings.json by default so you can have a clean install.
